public/css/style.css

```diff
@@ -170,7 +170,7 @@ input[type=reset] {
 -------------------------------------------------------------------------------------------------*/
 .footer {
 	background-color: #222;
-	height: 15vh
+	height: 15vh;
```

Comment:
is there any specific reason to use vh for height?

Reply:
I did not make the footer class style sheet i only did the css style1